[2022-04-13 22:33:33,770] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:20+00:00 [queued]>
[2022-04-13 22:33:33,805] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:20+00:00 [queued]>
[2022-04-13 22:33:33,806] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:33:33,806] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:33:33,807] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:33:33,853] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-02-02 06:00:20+00:00
[2022-04-13 22:33:33,875] {standard_task_runner.py:52} INFO - Started process 10178 to run task
[2022-04-13 22:33:33,886] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-02-02T06:00:20+00:00', '--job-id', '1698', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpaoia9u13', '--error-file', '/tmp/tmpo7f16k16']
[2022-04-13 22:33:33,889] {standard_task_runner.py:77} INFO - Job 1698: Subtask ingest
[2022-04-13 22:33:34,129] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:20+00:00 [running]> on host 13db72185839
[2022-04-13 22:33:34,338] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-02-02T06:00:20+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-02T06:00:20+00:00
[2022-04-13 22:33:34,340] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:33:34,343] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:33:34,395] {subprocess.py:85} INFO - Output:
[2022-04-13 22:33:34,414] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:33:34,415] {subprocess.py:89} INFO - dags
[2022-04-13 22:33:34,417] {subprocess.py:89} INFO - logs
[2022-04-13 22:33:34,418] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:33:34,421] {subprocess.py:89} INFO - plugins
[2022-04-13 22:33:34,423] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:33:34,423] {subprocess.py:89} INFO - scripts
[2022-04-13 22:33:34,425] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:33:34,426] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:33:34,530] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220202T060020, start_date=20220413T223333, end_date=20220413T223334
[2022-04-13 22:33:34,604] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:33:34,990] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
